cub/test/catch2_test_device_scan.cu

```diff
@@ -210,6 +210,47 @@ CUB_TEST("Device scan works with all device interfaces", "[scan][device]", full_
     }
   }
 
+// To call DeviceScan::InclusiveScan with initial value, the cudaStream_t argument needs
+// to be passed explicitly in order to maintain backwards compatibility. Since
+// DECLARE_LAUNCH_WRAPPER passes stream by default as the last argument for TEST_LAUNCH = 2
+// we need to skip these set of tests.
+#if TEST_LAUNCH != 2
+  SECTION("inclusive scan with init value")
+  {
+    using op_t    = cub::Sum;
+    using accum_t = cub::detail::accumulator_t<op_t, input_t, input_t>;
+
+    // Scan operator
+    auto scan_op = unwrap_op(reference_extended_fp(d_in_it), op_t{});
+
+    // Prepare verification data
+    c2h::host_vector<input_t> host_items(in_items);
+    c2h::host_vector<output_t> expected_result(num_items);
+    compute_inclusive_scan_reference(
+      host_items.cbegin(), host_items.cend(), expected_result.begin(), scan_op, accum_t{});
+
+    // Run test
+    c2h::device_vector<output_t> out_result(num_items);
+    auto d_out_it = thrust::raw_pointer_cast(out_result.data());
+    using init_t  = cub::detail::value_t<decltype(unwrap_it(d_out_it))>;
+    cudaStream_t stream{};
+
+    device_inclusive_scan(unwrap_it(d_in_it), unwrap_it(d_out_it), scan_op, num_items, stream, init_t{});
+
```

Comment:
critical: the binary operator is sum, which makes this initial value an identity element. If this overload doesn't do with the initial value, we'll never see that. Let's follow what other tests do with respect to initial value.